styles/canary.json

```diff
@@ -70,6 +70,20 @@
 	},
 	"styles": {
 		"blocks": {
+			"core/comment-content": {
```

Comment:
This doesn't seem to work for me, these styles are being overridden (by the default link styles I think) and I'm seeing no underline by default. 

<img width="678" alt="image" src="https://user-images.githubusercontent.com/1645628/194298689-c8ff71ca-ad87-4592-8182-71ad78694ad5.png">

Swapping the block to `core/comments` seems to fix this.

```suggestion
			"core/comments": {
```

Reply:
It is intended for the comment content -Links in text that the comment author adds. That is the area that is covered by the accessibility-ready requirement. But that doesn't seem to be working either.

Reply:
Adding it to core/comments adds it to all links:
<img width="694" alt="Screen Shot 2022-10-06 at 13 56 09" src="https://user-images.githubusercontent.com/7422055/194306454-a10a5337-8d6d-4da4-9ec5-6f6e01bade2a.png">

Reply:
Yeah they didn't seem to work for me either, but they were included when I changed the block to `core/comments`.